handbook/engineering/code-intelligence/index.md

```diff
@@ -12,6 +12,20 @@ The Code Intelligence team builds tools and services that provide contextual inf
 
 - Robust, extensible, and scalable infrastructure to index code across all languages, keep those indexes up-to-date, and efficiently resolve code intelligence queries against all indexed code.
 
+## History
+
+The origin of this team is the core of Sourcegraph, the first iteration was [srclib](https://github.com/sourcegraph/srclib) a code analysis tool developed by our co-founders [Beyang](https://github.com/beyang) and [Quinn](https://github.com/sqs) in 2015.
+
+2016 - We evolved our strategy to utilize LSP (Language Server Protocol) to power the new Sourcegraph code search engine. LSP is a user oriented protocol and requires an implementation of a language server for each programming language.
+
+2017 - Over time we started to experience some issues with LSPs; they can be difficult to deploy, slow at runtime, slow community adoption, and slow to develop!
```

Comment:
Since this isn't an event or change maybe you should move this text closer to what we did about feeling that pain.

Reply:
You're right that it isn't an event or change but it was an experience or learning process and I wanted to highlight that.